src/index.js

```diff
@@ -1,29 +1,21 @@
 import ButtonSource from './components/Button'
 import CardSource from './components/Card'
-import CheckboxSource from './components/Checkbox'
 import ErrorSource from './components/Error'
 import HeadingSource from './components/Heading'
 import IconSource from './components/Icon'
-import InputSource from './components/Input'
 import ListSource from './components/List'
 import MarkdownSource from './components/Markdown'
 import ParagraphSource from './components/Paragraph'
-import RadioGroupSource from './components/RadioGroup'
 import RequestSource from './components/Request'
 import ToggleSource from './components/Toggle'
-import TooltipSource from './components/Tooltip'
 
 export const Button = ButtonSource
 export const Card = CardSource
-export const Checkbox = CheckboxSource
 export const Error = ErrorSource
 export const Heading = HeadingSource
 export const Icon = IconSource
-export const Input = InputSource
 export const List = ListSource
 export const Markdown = MarkdownSource
 export const Paragraph = ParagraphSource
-export const RadioGroup = RadioGroupSource
 export const Request = RequestSource
 export const Toggle = ToggleSource
-export const Tooltip = TooltipSource
```

Comment:
In the future, we wont build components directly in `egghead-ui`; instead, components are only "promoted" to this repo after they have been battle tested in an egghead project and would be useful to use across multiple projects